src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserStateObject.netfx.cs

```diff
@@ -377,7 +383,13 @@ private void ReadSniError(TdsParserStateObject stateObj, uint error)
 
         private uint GetSniPacket(PacketHandle packet, ref uint dataSize)
         {
-            return SniNativeWrapper.SniPacketGetData(packet, _inBuff, ref dataSize);
+            return SniPacketGetData(packet, _inBuff, ref dataSize);
+        }
+
+        private uint SniPacketGetData(PacketHandle packet, byte[] _inBuff, ref uint dataSize)
```

Comment:
To me this seems unnecessary - just put the contents of this method into GetSniPacket. But if you made this change because netcore does it this way, I'd say we should make netcore do it the simpler way.

Reply:
This needs to remain as-is, I'm afraid. In netcore, SniPacketGetData is an abstract method. This change syncs up the method definitions so they can be merged in #3355 - commit is [here](https://github.com/dotnet/SqlClient/pull/3355/commits/e06dba95519f717c59e207a174f9f5cb85b8ceb2).

Reply:
Ok, i think that's ok for now